### DISCUSSION TITLE
Execute commands in Ubuntu

### DISCUSSION THREAD

**Discussion by Flyslinger2** [strongswan/strongswan#2120] [General]
# pki --scepa 
Error: invalid operation
#pki --scep
Error: invalid operation 

I'm executing these commands based on this documentation:
https://docs.strongswan.org/docs/5.9/pki/pkiScepCa.html
https://docs.strongswan.org/docs/5.9/pki/pkiScep.html

Enlighten me as to what I am doing wrong, please.

**Comment by tobiasbrunner** [comment]
Please read the documentation, these commands were added with 5.9.8. Ubuntu does not ship that version yet.
